---
layout: archive
title: ""
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

EDUCATION
======

**Sun Yat-Sen University**
*Aug. 2021 - Present
*Master in Science
*Major: 	Ecology (honors program: Evolutionary Ecology)
*Cumulative GPA:	3.5 (85.65/100)
*Core Courses: 	Evolutionary Ecology, Seminar in Ecology and Evolution, Data Analysis in R

**Fujian Agriculture and Forestry University**
*Sep. 2017 - Jun. 2021
*Bachelor of Agriculture, summa cum laude
*Majors: 			Plant Science and Technology
*Cumulative GPA:		3.4/4
*Relevant Courses:	Microbiology, Genetics (Bilingual 	Course), Molecular Genetics, Biological 	Data Analysis, Organic Chemistry B, 	Plant Physiology A, Biochemistry B, 	Probability Theory, Experimental 	Designs and Statistical Analyses, Plant 	Cell Biology, Agroecology, Plant 	Biotechnology	

RESEARCH EXPERIENCE
======

* 1.**Crabs and Symbiotic Microorganisms: Collaborative Cellulose Degradation Facilitating Radiative Diversity in _Sesarmidae_ — Genomic and Gut Microbiome Macro-genomic Analysis of Crabs**
* Nov. 2022 - Present
**Co-first author**, in progress, in collaboration with The Chinese University of Hong Kong	
*	Led the assembly of three Sesarmidae genomes.
*	Conducted gene functional annotations, focusing on CAZymes, using public databases. 
*	Analyzed genomic collinearity, evaluated CAZymes families, and identified unique gene families and expansions/contractions. 
*	Combined metagenomics and metatranscriptomics to uncover crab lignocellulose degradation mechanisms and evolution.


* 2	**Terrestrial Adaptation Evolution in Land Crabs: Exploring at the Genomic Level** &emsp; &emsp; Shenzhen, China
* **First author**, Upcoming **Graduation Thesis** &emsp; &emsp; &emsp; &emsp;  &emsp;  &emsp; &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Sep. 2022 - Present
* 	Completed **genome assembly** using third-generation ONT sequencing data and Hi-C data.
* 	Individually extracted RNA from five tissues of the Land Crabs (_Gecarcoidea lalandii_), constructed **RNA libraries**, and performed **independent** third-generation **full-length cDNA ONT sequencing**.
* 	Conducted **gene functional annotation** amalgamating second and third-generation transcriptomic data and homologous protein information, utilizing the Maker tool and public database.
* 	Performed inner **genomic collinearity analysis, unique and shared gene family assessment**, and **positive selection analysis** to identify distinctive gene families, significant expansions, and contractions in _Gecarcoidea lalandii_.
Additionally, scrutinized for _genes under positive selection_ in _Gecarcoidea lalandii_, with the objective of elucidating the driving factors behind its adaptive evolution (_terrestrial adaptation_).

* 3	**Uncovering Influenza-Like Virus Clades and New Genera in Invertebrates: Evolutionary Insights into Orthomyxoviridae across Metazoans** &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; Hongkong, China
* **Co-first author**, In submission &emsp; &emsp; &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Jun. 2022 - Jul. 2023
* 	**Novel Orthomyxoviridae identified.** Constructed phylogenetic relationship of newly identified viruses with known viral members from Orthomyxoviridae.
* 	**Ancestral states of the novel viruses and host associations analysis.**

* 4	[Transcriptome Analysis of Pathogen-Induced Physiological Responses in Shrimp](https://www.mdpi.com/2077-1312/11/2/389)  &emsp; &emsp; Shenzhen, China
* First author, Published&emsp; &emsp; &emsp; &emsp;  &emsp;  &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp; &emsp;  &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Feb. 2022 - Jan. 2023
* 	Gathered _nine RNA-Seq project datasets_ from the NCBI SRA and finally obtained _109 transcriptome expression profiles_.
* 	Generated **innovative ideas** to enhance project outcomes.
* 	Proficiently acquired expertise in **Shell scripting** and effectively utilized **R programming languages**, which enabled me to complete the Data Processing, Plot Generation, and Initial Draft Writing independently.

* 5	[Microscopic Algae, "Macroscopic" Energy](https://2021.igem.org/Team:CHINA-FAFU)  &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Fuzhou, China
* Student Leaders & Advisors, iGEM Project &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Jun. 2018 - Nov. 2021
* * 	Led the construction of transgenic algal strains to enhance carbon dioxide fixation and lipid synthesis metabolism by overexpressing critical genes in the ascorbic acid metabolic pathway.
* * 	Led mathematical modeling experiments, **including**
* * 	**Designing** a [Logistic growth model](https://2021.igem.org/Team:CHINA-FAFU/Model)  for microalgae based on _growth trends_.
* * 	**Developed** [optimization models](https://2021.igem.org/Team:CHINA-FAFU/Model)  using cost and time data to _predict optimal cultivation conditions_ and evaluate microalgae factory _cost-effectiveness and profitability_.

ACADEMIC ACTIVITIES AND PUBLICATION
======
	[1] Wu ZW, Chu KH*, Ma KY*. Transcriptome Analysis of Multiple Tissues in the Shrimp Penaeus vannamei Reveals the Typical Physiological Response to Three Pathogens[J]. Journal of Marine Science and Engineering. 2023; 11(2):389.
	[2] Wu ZW, Chu KH, Ma KY. Transcriptome analysis of multiple tissues of Penaeus vannamei reveals the typical physiological response to the invasion of three pathogens, THE 16th SYMPOSIUMOF CRUSTACEAN SOCIETY, 12-13 Nov 2022, Heibei Province, China (oral presentation).
	[3] Wu ZW, Chu KH, Ma KY. Transcriptome analysis of multiple tissues of Penaeus vannamei reveals the typical physiological response to the invasion of three pathogens, Dialogue with fantastic creatures—SYSU&HKU paleontology theme workshop, 15-17 Oct 2022, Guangdong Province, China (oral presentation).

ADDITIONAL EXPERIENCE
======
* 1	Undergraduate Ecological Comprehensive Experiment	&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Mar. 2023 - Jun. 2023 
* Laboratory Assistant in the 2022 spring semester &emsp; &emsp; &emsp; &emsp; &emsp;High-Throughput Sequencing Module
* 2	Undergraduate Biochemical Experiment &emsp; &emsp; &emsp;  &emsp;  &emsp; &emsp;  &emsp;  &emsp;  &emsp; &emsp; &emsp; Oct. 2022 - Jan. 2023 
* Laboratory Assistant in the 2022 autumn semester &emsp; &emsp; &emsp; &emsp; Biochemical Experiment
* 3	English - Chinese translation of a sustainable development report &emsp; &emsp; Oct. 2022 - Nov. 2022 
* United Nations Volunteers &emsp; &emsp; &emsp; &emsp; &emsp;  &emsp;  &emsp;  &emsp; &emsp; &emsp;  &emsp;  &emsp;  &emsp;  &emsp; &emsp; &emsp; &emsp; Translator
* 4	Undergraduate Ecological Comprehensive Experiment &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Mar. 2022 - Jun. 2022 
* Laboratory Assistant in the 2021 autumn semester &emsp; &emsp; &emsp; &emsp; High-Throughput Sequencing Module


  
SKILLS AND HOBBIE
======
* **Software:** Proficient in R (Preferred for analysis and plotting), Strong command of Shell programming (Primary tools), Familiar with Python (Supplementary tools), Limited exposure to Perl
* **Hobbies:** Avid Reader & Passionate Cook & Food Enthusiast & Animal lover

  
